addon/modifiers/sortable-item.js

```diff
@@ -268,6 +268,8 @@ export default class SortableItemModifier extends Modifier {
       return;
     }
 
+    this.setupHandleElement();
```

Comment:
This is the main change. If you run the demo and comment out this line, you will see the error I reported in the issue.